software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,5 +105,5 @@
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
     
     AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrmPort");
+    AttributeSensorAndConfigKey<Integer, Integer> WINRM_PORT = WinRmTool.PROP_PORT;
```

Comment:
The port is a constant in this code which is retrieved by the location config.
It is not a trivial case where an entity should allocate the next available port.
I do not think this change is needed.